test/models/tf_ov_model_zoo/run_infer_multiple.sh

```diff
@@ -24,11 +24,32 @@ while getopts “:m:b:d:h” opt; do
 done
 shift $((OPTIND-1))
 
+# Display link in BuildKite CI Web
+function inline_link {
+  LINK=$(printf "url='%s'" "$1")
+  if [ $# -gt 1 ]; then
+    LINK=$(printf "$LINK;content='%s'" "$2")
+  fi
+  printf '\033]1339;%s\a\n' "$LINK"
+}
+
+# Display image in BuildKite CI Web
+function inline_image {
+  printf '\033]1338;url='"$1"';alt='"$2"'\a\n'
+}
```

Comment:
This seems to be used. Should we display the plots inline and have links to the CSV in the log output?

Reply:
Deleting unused funcs. Still having path/location issues to display inline image